apps/src/netsim/NetSimTable.js

```diff
@@ -217,8 +210,7 @@ NetSimTable.prototype.subscribe = function () {
  * local channel. Also clears the saved callback.
  */
 NetSimTable.prototype.unsubscribe = function () {
-  this.channel_.unsubscribe(this.tableName_, this.channelCallback_);
-  this.channelCallback = undefined;
+  this.channel_.unsubscribe(this.tableName_);
```

Comment:
Did update comments to remove references to 'saved callback' since it's no longer a concern at this layer; other issues addressed in PubSubChannel.